whois-rdap/src/main/java/net/ripe/db/whois/rdap/RdapController.java

```diff
@@ -59,7 +59,7 @@ public class RdapController {
     private final String baseUrl;
     private final int maxResultSize;
     private final RdapRelationService rdapRelationService;
-
+    private final SourceContext sourceContext;
```

Comment:
Every Rdap query should go to slave datasource so I think we should fix it by changing the RdapRelationService to inject JdbcRpslObjectSlaveDao instaed of JdbcRpslObjectDao